_posts/2022-05-18-animated-scroll-effects.md

```diff
@@ -0,0 +1,136 @@
+
+The [intersection observer API](https://developer.mozilla.org/en-US/docs/Web/API/Intersection_Observer_API) provides a vanilla JavaScript approach to adding scroll effects. by triggering functions when elements intersect certain points of the viewport. Intersection observer can be difficult to work with, so a library such as [Animate on scroll](https://michalsnik.github.io/aos/) may make things simpler. 
+
+My personal favorite library to animate with is the [GreenSock Animation Platform](https://greensock.com/). GSAP is useful for all sorts of animations, including scroll triggered effects. 
+
+To show this off, let’s build a neat effect I came across on a website recently! The website included a hero section containing text. As you scroll the page, the text scales to highlight the importance of the message.
+
+For this demo I have created a [Codepen](https://codepen.io/mrtrimble/pen/gOvLPgM??editors=0010) to follow along with. I already included HTML, CSS, GSAP, and the ScrollTrigger plugin, so no need to setup anything! We’ll focus on the JavaScript to find out that with a few lines of code, we can build a neat scroll trigger animation.
+
+### Demo
+One of the key things GSAP provides as a library is the ability to create animations based on a **timeline**. This enables much more complex animations than what CSS keyframes provide. To create a GSAP timeline, we can initialize like so:
+
+```js 
+const tl = new gsap.timeline();
+```
+
+Now we are able to describe what animations should occur within that timeline - we can do that with **tweens**. Tweens explain what the animation should do be**tween** the start and end of the timeline. 
+
+Tweens take in two key parameters:
+- The element you want to animate.
+- An object explaining how the element will animate, including CSS properties to animate.
+
+You can include as many tweens as needed to complete the animation. GSAP includes three different types of tweens:
+- `.to()` - animates an element *to* something.
+- `.from()` - animates an element *from* something.
+- `.fromTo()` - animates an element *from* one thing *to* another.
+
+Pretty straightforward! To create tweens, we can tack them onto the timeline. To make things even easier, we can chain tweens together.
+
+So in our example, we want to create a tween that scales the `.scale-text` element to a large size:
+
+```js
+const tl = new gsap.timeline();
+
+tl.to('.scale-text', {
+  scale: 10
+});
+```
+
+The animation should trigger, however that is not quite what we want yet - we want this to happen on scroll. This is where the GSAP ScrollTrigger plugin comes in!
+
+Inside where we initialized our timeline, we can configure ScrollTrigger:
+
+```js
+const tl = new gsap.timeline({
+  scrollTrigger: {
+    trigger: ".hero",
+    start: "top top",
```

Comment:
I'd suggest using single quotes for this bit since you use single quotes above. Just kind of setting consistency